techmatters/flex-plugins
# Address styling diffs between Twilio & Aselo Messages List

## Description
This PR:
- Reworks styles for AseloMessageList (message list replacement), to make them look like the Flex 2 styles.
- Fixes a small bug where sending a message won't revert the text area to it's initial height.
- Adds debouncing to the "resize text area" behavior, as I've seen some degradation when not doing so:
  - No debounce:
    ![Screenshot from 2023-04-26 21-18-12](https://user-images.githubusercontent.com/15805319/234681164-f9ae366f-bf28-4291-95d1-db97a7ef8b96.png)
  - 300ms debounce:
    ![Screenshot from 2023-04-26 21-22-01](https://user-images.githubusercontent.com/15805319/234681159-f51d5474-ce71-4f83-970f-6b1cb0772aea.png)
  - 500ms debounce:
    ![Screenshot from 2023-04-26 21-14-06](https://user-images.githubusercontent.com/15805319/234681168-b6a73926-f58f-4524-b853-af509b6472ae.png)

Screenshots of the changes (what to look at):
- Transcript of a saved contact:
  ![Screenshot from 2023-04-26 20-07-39](https://user-images.githubusercontent.com/15805319/234681176-b5c65ae3-c269-45c1-897f-b11c4d92fb7f.png)
- Message list in a live chat
  ![Screenshot from 2023-04-26 20-05-01](https://user-images.githubusercontent.com/15805319/234681182-90c782ed-ff1f-4ad5-814a-817d9832d8e2.png)
  ![Screenshot from 2023-04-26 20-04-52](https://user-images.githubusercontent.com/15805319/234681184-a414bdec-c68f-47eb-bf81-3f7fe800365f.png)


### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1897)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for call contacts

### Verification steps
- Enable the feature flag, can be done via `src/hrmConfig.ts` by returning
```
featureFlags: {
  ...featureFlags,
  enable_aselo_messaging_ui: true,
},
```
- `npm run dev`  to start local server.
- Start and accept a new chat based contact.
- Confirm that the styles are looking like those in default Flex setup (or in the screenshots). Ideally test this in different monitor sizes to confirm nothing goes crazy.
